plugins/org.komodo.core/src/config/komodo.cnd

```diff
@@ -56,9 +56,6 @@
  * data sources, and other resources created and being worked on by the user.
  */
 [tko:workspace] > nt:unstructured
-  + tko:repositories (tko:repositories) copy
```

Comment:
@elvisisking 
We always would control the location of these things by the logic of the code when a user calls 'create teiid connection'. They never have access to the repository unless they use some external client, in which case having the app assume a fixed location for these artifacts becomes even more problematic. Using the search API and finding 'all teiid connections under workspace' is a lot more flexible, ditto 'all teiid connections under library'.